uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1020.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1020 {
+
+  /**
+   * Removes duplicated "Localise CiviHR" menu item
+   *
+   * This upgrader will:
+   * - Delete any menu item with the "Localise CiviHR" label
+   * - Makes sure that one menu item with the "Localise CiviCRM" label exists
+   *
+   * On the interface, due to the word replace in the HRUI extension, the
+   * "Localise CiviCRM" label will be displayed as "Localise CiviHR".
+   */
+  public function upgrade_1020() {
+    $this->up1020_deleteLocaliseCiviHRMenuItem();
+    $this->up1020_createLocaliseCiviCRMMenuItemIfItDoesNotExist();
+
+    return TRUE;
+  }
+
+  /**
+   * Deletes a menu item with the "Localise CiviHR" label, if it exists.
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function up1020_deleteLocaliseCiviHRMenuItem() {
+    civicrm_api3('Navigation', 'get', [
+      'sequential' => 1,
+      'label' => 'Localise CiviHR',
+      'api.Navigation.delete' => ['id' => '$value.id'],
+    ]);
+  }
+
+  /**
+   * Creates a new menu item with the "Localise CiviCRM" label, if not such
+   * item exists.
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function up1020_createLocaliseCiviCRMMenuItemIfItDoesNotExist() {
+    $domain = CRM_Core_Config::domainID();
+    $params = ['return' => 'id', 'name' => 'Administer', 'domain_id' => $domain];
+    $administerId = (int) civicrm_api3('Navigation', 'getvalue', $params);
+
+    if (!$administerId) {
```

Comment:
I might be wrong, but if you do want to handle this case then I think this won't work as `getvalue` will throw an exception if it doesn't find a single match